### ISSUE TITLE
package-lint version stuck?

### ISSUE THREAD

**Issue by alphapapa** [melpa/melpa#8442]
Forgive me if I'm missing something, but I couldn't find any issues on the MELPA repo nor the package-lint one about this problem.

The most recent commit on the package-lint repo is https://github.com/purcell/package-lint/commit/cc71fa62db9a7328ca055430c6f89b4f9a646c7b from Feb 15, 2023, but the version currently offered by MELPA is 20221003.1636.

Is something stuck?  @purcell @milkypostman 

Thanks.

**Comment by milkypostman** [comment]
I will take a look tomorrow. Thanks for the report!

**Comment by tarsius** [comment]
We only create a new snapshot if a relevant file is modified in the new commits. If no file that is included in the package is modified, then we don't create a new snapshot because it would be identical to the previous snapshot.

**Comment by alphapapa** [comment]
Indeed, I knew that, but I wrongly assumed that the updates to the package since October were to the code.  Sorry for the noise.
